plugin/eqdetect/source/dextool/plugin/eqdetect/subfolder/codegenerator.d

```diff
@@ -51,4 +51,48 @@ class SnippetFinder{
         temp = temp ~ content[mutation.offset_end .. content.length];
         return temp;
     }
+
+    @trusted static auto generateKlee(string[] params, string source_name, string mutant_name, string function_name){
+        import std.stdio;
+        import std.format;
+        //string code;
+        import dsrcgen.c;
+        auto code = new CModule();
+        //add klee imports
+        code.include("<klee/klee.h>");
+        code.include("<assert.h>");
+
+        //add import for files under test
+        code.include(format("\"%s\"", source_name));
```

Comment:
tip: in D it is possible to use the backtick too. It usually avoids these escaping.

```d
format(`foo "%s"`, source_name);
```